requirements.txt

```diff
@@ -0,0 +1,5 @@
+boto3
```

Comment:
Can we lockdown the version of these? Boto3 periodically gets breaking changes published, and I would hate to pick up one of those by accident...